.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,16 @@
 
 ## This introduces a breaking change
 
-- [ ] Yes
-- [ ] No
+- Yes
+- No
 
 <!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->
 
+## This should yield a new module release
+
```

Comment:
I would suggest to add an HTML comment with a brief explanation, similar to other parts of the template

Reply:
Thanks, this is now changed by https://github.com/thoth-station/template-project/pull/31/commits/eb07ab8ebd27eb198dcc06d288062a84e46d0594